README.md

```diff
@@ -57,9 +57,10 @@ Texera is a system to support collaborative, ML-centric data analytics as a clou
 
 ## Sample Workflow
 
-The following is a workflow formulated using the Texera GUI in a Web browser, which consists of operators such as regex search, sentiment analysis, user-defined function (UDF) in Python, and visualization.
+The following workflow, created using the Texera GUI in a web browser, focuses on jail data analysis. It includes data preparation, ML model training, and validation.
```

Comment:
The screenshot contains more than just a sample workflow. Give the caption and even the section name a bigger scope.

Reply:
updated.